src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -81,6 +83,12 @@ export class SkyAvatarComponent {
   @Input()
   public maxFileSize = 500000;
 
+  /**
+   * Specifies the size of the avatar. Default is "large".
+   */
+  @Input()
+  public size: 'large' | 'medium' | 'small' = 'large';
```

Comment:
Should we create a type for this, since its re-used in a couple places?

Reply:
Fixed.